lua/nvim-tree/renderer/components/diagnostics.lua

```diff
@@ -72,7 +72,7 @@ function M.setup(opts)
   }
   M.ICON[vim.diagnostic.severity.INFO] = {
     str = M.config.diagnostics.icons.info,
-    hl = { "NvimTreeLspDiagnosticsInfo" },
+    hl = { "NvimTreeLspDiagnosticsInformation" },
```

Comment:
It's a minor gripe, but I would do it the other way around, replacing the others *Information with *Info, since the level is `info` it would be more intuitive

Reply:
Agreed. `Info` is the consistent option.

I'm just putting this quick fix in for now. I've updated them all to `Info` in the long running feature branch: https://github.com/nvim-tree/nvim-tree.lua/pull/2455/files#diff-b5b2d1abcb86ead5a2319f7e065a7545b834a407cc9f420590d7acebb285fca9R2283